Packs/Lastline/Integrations/Lastline_v2/Lastline_v2_test.py

```diff
@@ -146,3 +147,27 @@ def test_test_module_wrong_credentials(mocker, params):
     client = Client(base_url=params.get('url'), api_params=api_params, credentials=params.get('credentials'))
     with pytest.raises(DemistoException, match='Authentication Error.'):
         client.test_module_command()
+
+
+@pytest.mark.parametrize('params', [{'url': 'testurl.com',
+                                     'credentials': {'identifier': 'identifier',
+                                                     'password': 'password'}}])
+def test_csvs_handles_correctly(mocker, params):
```

Comment:
Improve function name.
The name should include the name of the unit it tests (command/function) and the use case.